### PR TITLE
Reduce unnecessary readme.md

### PR DESCRIPTION
**Pull request by hungry1526**
Resolve issue 384:

1) delete unnecessary readme.md while going through the repo
2) too many readme.md is hard to maintain, therefore removing
3) remove empty space and newline 
4) simplify bug/issue template

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,10 +53,10 @@ Done predicting. Predict table : iris.predict
 - [Installation](doc/installation.md)
 - [Running a Demo](doc/demo.md)
 - [Extended SQL Syntax](doc/syntax.md)
+- [Build from source code](doc/build.md)
```
Comment (wangkuiyi) [inline README.md]: Users don't need to build from source, only contributors need to.

File: sql/python/README.md [hunk 2]
```diff
@@ -1,13 +0,0 @@
-# Python Code Template
-
```
Comment (wangkuiyi) [inline sql/python/README.md]: I agree that we should merge the content of this file into build from source doc. but I am not sure the docker command in this file runs now. I'd suggest we confirm it works before moving it into the build from source file

File: sql/python/README.md [hunk 3]
```diff
@@ -1,13 +0,0 @@
-# Python Code Template
```
Comment (wangkuiyi) [inline sql/python/README.md]: I am afraid this file is not ready to merge into the build from source file

**Review by wangkuiyi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (hungry1526) [reply README.md]: Users and developers seem be inter-changable,  I agree we can keep it under contribution.

File: sql/python/README.md (hunk 2 above)
Comment (hungry1526) [reply sql/python/README.md]: Will create an issue to track changes. For now, I will undo delete.

**Comment by hungry1526** [comment]
All comments are resolved.

**Review by wangkuiyi — APPROVED** [review]

**Pull request merged**